test/Bucket/albedo_map_bucket.jl

```diff
@@ -0,0 +1,139 @@
+using Test
+
+using ClimaCore
+using ClimaComms
+import CLIMAParameters as CP
+
+if !("." in LOAD_PATH)
+    push!(LOAD_PATH, ".")
+end
+using ClimaLSM.Regridder: MapInfo, regrid_netcdf_to_field
+using ClimaLSM.Bucket:
+    BucketModel,
+    BucketModelParameters,
+    PrescribedAtmosphere,
+    PrescribedRadiativeFluxes,
+    BulkAlbedoMap,
+    cesm2_land_albedo_dataset_path
+using ClimaLSM.Domains:
+    coordinates, LSMSingleColumnDomain, LSMSphericalShellDomain
+using ClimaLSM:
+    initialize, make_update_aux, make_ode_function, make_set_initial_aux_state
+
+# Bucket model parameters
+import ClimaLSM
+include(joinpath(pkgdir(ClimaLSM), "parameters", "create_parameters.jl"))
+
+@testset "Initialize aux state with map" begin
+    FT = Float32
+    earth_param_set = create_lsm_parameters(FT)
+
+
+    α_snow = FT(0.8) # snow albedo
+    path = cesm2_land_albedo_dataset_path()
```

Comment:
It might be nice to have an additional `MapInfo` constructor which has default values for all the arguments, so a basic map can be created without having to know about the internals. The coupler has to create an albedo `MapInfo` struct and I think it would be better if it didn't have to use e.g. `cesm2_land_albedo_dataset_path()`. This could be a separate small PR, or maybe part of this one.

Reply:
ok! I think the default constructor would need to call cesm2_land_albedo_dataset_path() - to get the local path of the file - but that's fine! Ill add this in today.

Reply:
ok! this is fixed. you can now call it with albedo_model = BulkAlbedoMap{FT}(regrid_dirpath). It also has a default value for alpha snow, and comms context, so you'll have to pass in the comms context from the coupler.
https://github.com/CliMA/ClimaLSM.jl/blob/6d4e3b11a7b407011df0d76570f5c10865f57f98/test/Bucket/albedo_map_bucket.jl#L35
https://github.com/CliMA/ClimaLSM.jl/blob/6d4e3b11a7b407011df0d76570f5c10865f57f98/src/Bucket/Bucket.jl#L100